coralproject/talk
# Fix CLI users create task

## What does this PR do?

This PR fixes the CLI `users create` task - its currently failing upon user creation due to a missing argument.

`TypeError: Cannot read property 'trim' of undefined
    at Function.createLocalUser (/Users/clint.brown/workspace/talk/services/users.js:580:25)
    at Command.createUser (/Users/clint.brown/workspace/talk/bin/cli-users:291:37)
    at <anonymous>`

## How do I test this PR?

- Run CLI `users create` task
- Complete the data entry steps and confirm the user is created successfully
